scripts/namespace_codegen_templates.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This script copies the oapi-codegen templates to a temporary directory and
+# renames/namespaces the import symbol of their "runtime" package.
+# This is necessary to avoid a name collision with our URL parameters called "runtime"
+# in the autogenerated code.
+#
+# To use the templates, run oapi-codegen with `-templates /tmp/namespaced-templates`.
+
+# Find the local directory of the oapi-codegen templates.
+codegenDir="$(go list -f "{{.Dir}}" github.com/deepmap/oapi-codegen/pkg/codegen)"
+[[ "$codegenDir" != "" ]] || { echo "cannot locate oapi-codegen templates dir"; exit 1; }
+
+# Create a working copy of the templates.
+rm -rf /tmp/namespaced-templates
+cp -r "$codegenDir/templates" /tmp/namespaced-templates
+find /tmp/namespaced-templates/ -type d -print0 | xargs -0 chmod 755
+
+# Rename the "runtime" package to "codegen_runtime" in all templates.
+find /tmp/namespaced-templates -type f -print0 | xargs -0 sed -i -E 's/\bruntime\b/codegen_runtime/g'
+# Fix the import statement to use the new name.
+sed -i -E 's!"github.com/deepmap/oapi-codegen/pkg/codegen_runtime"!codegen_runtime "github.com/deepmap/oapi-codegen/pkg/runtime"!' /tmp/namespaced-templates/imports.tmpl 
```

Comment:
🤯 

(going to assume this works if `make oasis-indexer` works 😄 )

Reply:
bash is a dangerous weapon :)
it's not as unreadable as it looks at first glance; the commands are actually simply, just the syntax is ugly.

it does work; I don't much worry about it breaking since we're using a fixed version of the codegen. However :exploding_head: is :no_good:, and I seriously considered not doing this and renaming `runtime` instead. But it's such a prominently displayed, outward-facing variable in URLs, and there's not a good alternative. `rtime`, `rt` ... too cryptic, inconsistent with other names. `paratime` ... not used in code (and the name does penetrate our Go code in many places).

Reply:
Makes sense, thanks for explaining the thought process